docs/index.md

```diff
@@ -1,49 +1,40 @@
-# Overview
+# About UCC
 
-`splunk-add-on-ucc-framework` is a framework to generate UI-based Splunk
-add-ons. It includes UI, REST handlers, Modular inputs, OAuth and Alert
-action templates.
+Universal Configuration Console (UCC) is a framework that simplifies the process of add-on creation for developers. You can use UCC to generate UI-based Splunk add-ons. UCC includes UI, REST handlers, modular inputs, OAuth, and alert action templates.
 
-Only add-ons that use Python 3 are supported.
+The UCC framework helps you to maintain consistency and a uniform look and feel across different add-ons. You can easily update and modify your add-ons.
 
-It is available as a GitHub action here:
-<https://github.com/splunk/addonfactory-ucc-generator-action>
+UCC 5 <!--- why do we say 5, everywhere else we say UCC---> uses Splunk UI. See [Splunk UI](https://splunkui.splunk.com/). It is a new UI framework based on React. The UCC UI repository is stored in the `ui` folder. <!-- (but where is this folder exactly, when I install Splunk UI?) -->
```

Comment:
That's a legacy name "UCC 5", this was a major release which started using Splunk UI and not old technology.

I think we can remove this sentence.

Reply:
Removed